packages/create-pwa/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/magento/pwa-studio/tree/master/packages/create-pwa#readme",
   "dependencies": {
-    "@magento/pwa-buildpack": "10.0.0-alpha.5",
+    "@magento/pwa-buildpack": "10.0.0-alpha.6",
```

Comment:
Our version bumps are automated in AWS, and I'm not sure if the jobs would break with these manual bumps. You'll likely need to undo all these, but was there a reason this was necessary?

Reply:
Force of habit of updating versions when we apply a modification. I'll remove it now.